src/us/kbase/workspace/test/kbase/JSONRPCLayerTest.java

```diff
@@ -1043,7 +1042,7 @@ public void saveAndGetObjects() throws Exception {
 		loi.add(new ObjectIdentity().withWsid(wsid).withName("auto2").withVer(1L));
 		loi.add(new ObjectIdentity().withWsid(wsid).withObjid(2L).withVer(1L));
 		checkSavedObjects(loi, 2, "auto2", SAFE_TYPE, 1, USER1,
-				wsid, "saveget", "36c4f68f2c98971b9736839232eb08f4", 23, meta, data);
+				wsid, "saveget", "36c4f68f2c98971b9736839232eb08f4", 23, meta, mtmeta, data);
```

Comment:
ZOMG!! How did you know that "36c4f68f2c98971b9736839232eb08f4" is my favourite string?!?!

Reply:
https://neuralink.com/